sdk/translation/ai-translation-document-rest/review/ai-translation-document.api.md

```diff
@@ -573,7 +566,7 @@ export interface StartTranslationLogicalResponse extends HttpResponse {
 export type StartTranslationParameters = StartTranslationBodyParam & RequestParameters;
 
 // @public
-export type StatusOutput = "NotStarted" | "Running" | "Succeeded" | "Failed" | "Cancelled" | "Cancelling" | "ValidationFailed" | string;
+export type StatusOutput = string;
```

Comment:
Why did we also remove the extensible enums here too?

Reply:
Emitter change